src/mmda/parsers/grobid_augment_existing_document_parser.py

```diff
@@ -61,61 +61,87 @@ def parse(self, input_pdf_path: str, doc: Document, xml_out_dir: Optional[str] =
         self._parse_xml_onto_doc(xml, doc)
         return doc
 
+    def _grobid_box_groups_to_span_groups(self, doc: Document, **kwargs: Iterable[BoxGroup]) -> Dict[
+        str, List[SpanGroup]]:
+        span_groups_by_field_name = defaultdict(list)
+        for parsed_field_name, box_groups in kwargs.items():
+            span_groups_by_field_name[parsed_field_name] = box_groups_to_span_groups(box_groups, doc, center=True)
+        return span_groups_by_field_name
+
     def _parse_xml_onto_doc(self, xml: str, doc: Document) -> Document:
         xml_root = et.fromstring(xml)
+        self._cache_page_sizes(xml_root)
 
-        bib_entry_box_groups = self._get_grobid_bib_box_groups(xml_root)
-
-        # get nice SpanGroups from tokens whose centers overlap with the Grobid box
-        bib_entry_span_groups = box_groups_to_span_groups(bib_entry_box_groups, doc, center=True)
+        box_group_parses = defaultdict(list)
 
+        # get raw box groups from Grobid XML output
+        box_group_parses["authors"] = self._get_author_box_groups(xml_root)
         # note for if/when adding in relations between mention sources and bib targets:
         # big_entries metadata contains original grobid id attached to the BoxGroup.
-        doc.annotate(
-            bib_entries=bib_entry_span_groups
-            )
+        box_group_parses["bib_entries"] = self._get_bib_entry_box_groups(xml_root)
+
+        # get nice SpanGroups from tokens whose centers overlap with the Grobid boxes
+        annotatable_span_groups_by_field_name = self._grobid_box_groups_to_span_groups(doc, **box_group_parses)
```

Comment:
this might just be my brain, but I think I would rather go field by field than make this nifty dict that kind of hides what you're actually adding to the doc. Is that possible? 

Something like:
```
author_span_groups = box_groups_to_span_groups(get_author_box_groups(xml_root))
bib_entry_span_groups = box_groups_to_span_groups(get_bib_entry_box_groups(xml_root))
doc.annotate(author_span_groups)
doc.annotate(bib_entry_span_groups)
```